src/main/java/com/ait/lienzo/client/core/shape/wires/layout/size/SizeConstraintsContainerLayout.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ait.lienzo.client.core.shape.wires.layout.size;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.ait.lienzo.client.core.shape.IPrimitive;
+import com.ait.lienzo.client.core.shape.wires.layout.AbstractContainerLayout;
+import com.ait.lienzo.client.core.shape.wires.layout.size.SizeConstraints.Type;
+import com.ait.lienzo.client.core.types.BoundingBox;
+import com.ait.tooling.common.api.java.util.function.BiFunction;
+
+public class SizeConstraintsContainerLayout extends AbstractContainerLayout<SizeConstraints>
+        implements IMaxSizeLayout<SizeConstraints>
+{
+    private static final Map<Type, BiFunction<SizeConstraints, BoundingBox, BoundingBox>> sizeBuilders = new HashMap<Type, BiFunction<SizeConstraints, BoundingBox, BoundingBox>>()
+    {{
```

Comment:
double-brace syntax, let's wait for @hasys to confirm if we should refactor this...

Reply:
:+1: